.github/workflows/test_integration_cli.yml

```diff
@@ -215,6 +221,8 @@ jobs:
             dbt: "1.2.*"
           - profile: "athena"
             dbt: "1.3.*"
+          - profile: "athena"
+            dbt: "1.4.*"
```

Comment:
Are we installing `dbt-athena-community`?

Reply:
No, this change can be done in a different PR.